src/benchmark_runner.cc

```diff
@@ -151,7 +155,7 @@ class BenchmarkRunner {
 
     for (int repetition_num = 0; repetition_num < repeats; repetition_num++) {
       const bool is_the_first_repetition = repetition_num == 0;
-      DoOneRepetition(is_the_first_repetition);
+      DoOneRepetition(is_the_first_repetition, repetition_num);
```

Comment:
i believe we don't need both `is_the_first_repetition` and `repetition_num`. We can derive the first from the second in `DoOneRepetition`.

Reply:
Fixed!